include/odp/api/abi-default/packet.h

```diff
@@ -72,6 +72,70 @@ typedef enum {
 
 #define ODP_NUM_PACKET_COLORS 3
 
+/** Parse result flags */
+typedef struct odp_packet_parse_result_flag_t {
+	/** Flags union */
+	union {
+		/** All flags as a 64 bit word */
+		uint64_t all;
```

Comment:
'all' can be used to clear all bit flags. It might be useful to group all errors to u8 or u16 to quick check if there any error.

Reply:
however current has_error also works here...